---
layout: default
title: Wang Lab
subtitle: Chenfei Wang Lab at Tongji University
use-site-title: true
---

<div class="container">
    <div class="row">
        <div id="carouselExampleIndicators" class="carousel slide carousel-fade" data-ride="carousel">
          <!-- <ol class="carousel-indicators">
            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
            <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
          </ol> -->
          <div class="carousel-inner">
            <div class="carousel-item active" style="height: 25rem">
              <img src="/assets/img/lab_2021.JPG" class="d-block w-100">
            </div>
<!--             <div class="carousel-item" style="height: 25rem">
              <img src="/assets/img/lab.jpg" class="d-block w-100">
            </div> -->
          </div>
          <!-- <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
            <span class="sr-only">Previous</span>
          </a>
          <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
            <span class="carousel-control-next-icon" aria-hidden="true"></span>
            <span class="sr-only">Next</span>
          </a> -->
        </div>
    </div>

    <div class="row" style="margin-top: 3rem">
        <h2>Research interests</h2>
        <p>Our research focuses on developing computational algorithms and conducting big data mining from high-throughput data to understand the mechanism of gene regulation and metabolic reprogramming in cellular status changes. We have developed a series of algorithms and web resources for single-cell transcriptomics, epigenomics, CRISPR screens, and spatial multi-omic data analysis and re-use. Through integrating multi-omics single-cell and spatial datasets, we try to decipher the transcriptional and epigenetic regulatory mechanisms underlying cellular status changes in cancer immunology, and developmental and aging-related diseases.
        </p>
    </div>
    <div class="row">
        <h3>Computational methods for single-cell and spatial multi-omics</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/STRIDE.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Single-cell multi-omics is important for understanding the heterogeneity of complex biological systems, however, there is a lack of effectively integrated analysis methods. We aim to develop machine learning algorithms based on single-cell and spatial transcriptomics and epigenomics data to systematically solve the problem of multi-omics data integration and analysis.  We have developed <a href="http://github.com/liulab-dfci/MAESTRO">MAESTRO</a> for integrative analysis of scRNA-seq and scATAC-seq data. <a href="https://github.com/wanglabtongji/STRIDE">STRIDE</a> for improving the resolution of Spatial Transcritomic data by integrating with scRNA-seq using topic modeling. Developed a scRNA-seq database <a href="http://tisch.comp-genomics.org">TISCH</a> for comprehensive visualization of the gene expression and cell-type composition in tumor microenvironment. Now we are working on multiple challenging questions in the single-cell field including automatic cell-type annotation, predicting and unraveling the gene regulation networks, improving the resolution and coverage of spatial transcriptomics, and inferring the cellular cross-talks and metabolic status.
                </p>
            </div>
        </div>
    </div>
    <div class="row">
        <h3>Mechanism in generating and remodeling immunosuppressive tumor microenvironment</h3>
        <div class="row">
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Cancer arises from the evasion of immune surveillance, and the immunosuppressive tumor microenvironment has a vital impact on tumor development and metastasis. We aim to integrate single-cell and spatial multi-omics data with comprehensive bioinformatics data analysis to investigate the effects of epigenetic remodeling and metabolic reprogramming in altering the immune cell status in the TME, and tried to develop potential immune remodeling methods for cancer treatment. We have been working closely with oncologists and immunologists to study the mechanism of carcinogenesis, immune and drug resistance in the head and neck, lung, and cervical cancer.
                </p>
            </div>
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/tumor.png">
            </div>
        </div>
    </div>
    <div class="row">
        <h3>Regulation of cell state transitions in tumor immunity, developmental and aging-related diseases</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/development.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Cell state alteration in a broad sense encompasses cell type differentiation, which plays an important role in the development, immune and aging-related diseases. Investigation between epigenetic remodeling and metabolic reprogramming will help to understand the molecular mechanisms of cell-type transition from both intrinsic and extrinsic perspectives, which could be further used to regulate and remodel the transition process. We have revealed the role of histone modifications such as H3K4me3, H3K27me3, H3K9me3, and nucleosome positioning in regulating cell fate decisions during early mammalian embryonic development, and demonstrated the function of broad H3K4me3 and lineage-specific H3K9me3 in cell-type differentiation (Nature 2016, Nat Cell Biol 2018, Cell Res 2022). We have also identified key epigenetic regulators in somatic nuclear-transferred embryo development (Cell Discov 2016, Cell Stem Cell 2018).
                </p>
            </div>
        </div>
    </div>
  </div>
</div>


