---
layout: default
title: Wang Lab
subtitle: Chenfei Wang Lab at Tongji University
use-site-title: true
---

<div class="container">
    <div class="row">
        <div id="carouselExampleIndicators" class="carousel slide carousel-fade" data-ride="carousel">
          <!-- <ol class="carousel-indicators">
            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
            <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
          </ol> -->
          <div class="carousel-inner">
            <div class="carousel-item active" style="height: 25rem">
              <img src="/assets/img/lab_2021.JPG" class="d-block w-100">
            </div>
<!--             <div class="carousel-item" style="height: 25rem">
              <img src="/assets/img/lab.jpg" class="d-block w-100">
            </div> -->
          </div>
          <!-- <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
            <span class="sr-only">Previous</span>
          </a>
          <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
            <span class="carousel-control-next-icon" aria-hidden="true"></span>
            <span class="sr-only">Next</span>
          </a> -->
        </div>
    </div>

    <div class="row" style="margin-top: 3rem">
        <h2>Research interests</h2>
        <p>Our research focuses on developing computational algorithms and conducting big data mining from high-throughput data to understand the mechanism of gene regulation and metabolic reprogramming. We have developed a series of algorithms and web resources for single-cell transcriptomics, epigenomic, CRISPR-screens and spatial multi-omic data analysis and re-use. Through integrating multi-omics single-cell and spatial datasets, we try to decipher the transcriptional and epigenetic regulatory mechanisms underlying human immune disorder, anti-tumor immunity, cellular and organ level differentiation.
        </p>
    </div>
    <div class="row">
        <h3>Computational methods for single-cell and spatial multi-omics</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/STRIDE.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">We have developed a series of computational algorithems for single-cell and spatial multi-omic data analyses. We have developed <a href="http://github.com/liulab-dfci/MAESTRO">MAESTRO</a> for integrative analysis of scRNA-seq and scATAC-seq data. <a href="https://github.com/wanglabtongji/STRIDE">STRIDE</a> for improving the resolution of Spatial Transcritomic data by integrating with scRNA-seq using topic modeling. Developed a scRNA-seq database <a href="http://tisch.comp-genomics.org">TISCH</a> for comprehensive visualization of the gene expression and cell-type composition in tumor microenvironment. Now we are working on multiple challenging questions in single-cell filed including automatic cell-type annotation, predicting and unraveling the regulation networks, improving the resolution of spatial transcriptomics into single-cell, and inferring the cellular cross-talks and metabolic status.
                </p>
            </div>
        </div>
    </div>
    <div class="row">
        <h3>Computational methods for single-cell and spatial multi-omics</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/STRIDE.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">We have developed a series of computational algorithems for single-cell and spatial multi-omic data analyses. We have developed <a href="http://github.com/liulab-dfci/MAESTRO">MAESTRO</a> for integrative analysis of scRNA-seq and scATAC-seq data. <a href="https://github.com/wanglabtongji/STRIDE">STRIDE</a> for improving the resolution of Spatial Transcritomic data by integrating with scRNA-seq using topic modeling. Developed a scRNA-seq database <a href="http://tisch.comp-genomics.org">TISCH</a> for comprehensive visualization of the gene expression and cell-type composition in tumor microenvironment. Now we are working on multiple challenging questions in single-cell filed including automatic cell-type annotation, predicting and unraveling the regulation networks, improving the resolution of spatial transcriptomics into single-cell, and inferring the cellular cross-talks and metabolic status.
                </p>
            </div>
        </div>
    </div>
    <div class="row">
        <h3>Mechanism in generating and remodeling immunosuppressive tumor microenvironment</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/tumor.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Cancer arises through evasion of immune killing. We are working on utilizing scRNA-seq, scATAC-seq, scCRISPR screens, and spatial multi-omics to deciphering the cellular heterogeneity and dynamics in the tumor microenvironment. We hope to discover intra- or inter-cellular mechanisms in generating immunosuppressive tumor microenvironment and develop novel strategies in remodeling the TME to cure cancer. We are working on multiple tumor models to investigate the immune evasion mechanism of cancer cells. We are also collaborating with oncologists to studying the mechanism of carcinogenesis, immune and drug resistance in the oral, lung, and cervical cancer.
                </p>
            </div>
        </div>
    </div>
    <div class="row">
        <h3>Epigenetics regulation of cell-fate decision in embryo and organ development</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/development.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">We aimed to investigate the gene regulation mechanisms in cell state transition and fate control, which could be used to improve the efficiency in rewiring cell fate. We have been studying the epigenetic inheritance and reprogramming during mammalian embryo development and its effect on embryogenesis and cell fate determination. We have revealed the dynamics of H3K4me3, H3K27me3, and H3K9me3 during mouse early embryo development, and demonstrated the function of broad H3K4me3 and lineage-specific H3K9me3 in cell-type differentiation (Nature 2016, Nat Cell Biol 2018). We have also identified key epigenetic regulators in somatic nuclear-transferred embryo development (Cell Discov 2016, Cell Stem Cell 2018).
                </p>
            </div>
        </div>
    </div>
  </div>
</div>


