---
layout: default
title: Wang Lab
subtitle: Chenfei Wang Lab at Tongji University
use-site-title: true
---

<div class="container">
    <div class="row">
        <div id="carouselExampleIndicators" class="carousel slide carousel-fade" data-ride="carousel">
          <!-- <ol class="carousel-indicators">
            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
            <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
          </ol> -->
          <div class="carousel-inner">
            <div class="carousel-item active" style="height: 25rem">
              <img src="/assets/img/lab_2021.JPG" class="d-block w-100">
            </div>
<!--             <div class="carousel-item" style="height: 25rem">
              <img src="/assets/img/lab.jpg" class="d-block w-100">
            </div> -->
          </div>
          <!-- <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
            <span class="sr-only">Previous</span>
          </a>
          <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
            <span class="carousel-control-next-icon" aria-hidden="true"></span>
            <span class="sr-only">Next</span>
          </a> -->
        </div>
    </div>

    <div class="row" style="margin-top: 3rem">
        <h2>Research interests</h2>
        <p>Our research focuses on developing computational algorithms and conducting big data mining from high-throughput data to understand gene regulation and cellular cross-talks in human immune and developmental systems. We have developed a series of algorithms and web resources for single-cell transcriptomics and epigenomic data analysis and re-use. We also working closely with immunologists and developmental biologists to decipher the transcriptional and epigenetic regulatory mechanisms underlying immune system aging and tumor immunity, human embryo development and differentiation.
        </p>
        <h3>Single-cell Algorithems and Resources</h3>
        <p>We have developed computational methods(<a href="http://github.com/liulab-dfci/MAESTRO">MAESTRO</a>) for integrative analysis of scRNA-seq and scATAC-seq data. We also developed a scRNA-seq database for comprehensive visualization of the gene expression and cell-type composition in tumor microenvironment(<a href="http://tisch.comp-genomics.org">TISCH</a>). Now we are working on multiple challenging questions in single-cell filed including automatic cell-type annotation, predicting and unraveling the regulation networks based on scATAC-seq data, and inferring the cellular crosstalks based on spatial transcriptomics data.
        </p>
        <h3>Immune system aging and tumor immunity</h3>
        <p>We are working on investigating the underlying mechanisms of tumor immunity and immune system aging based on scRNA-seq, scATAC-seq, and CRISPR screens. We are now collaborating with immunologists and cancer biologists to studying the mechanism of carcinogenesis, immune and drug resistance in oral and lung cancer.
        </p>
        <h3>Embryo development and differentiation</h3>
        <p>We have been studying the epigenetic inheritance and reprogramming during mammalian embryo development and its effect on embryogenesis and cell fate determination. We have revealed the dynamics of H3K4me3, H3K27me3, and H3K9me3 during mouse early embryo development, and demonstrated the function of broad H3K4me3 and lineage-specific H3K9me3 in cell-type differentiation (Nature 2016, Nat Cell Biol 2018). We have also identified key epigenetic regulators in somatic nuclear-transffered embryo development (Cell Discov 2016, Cell Stem Cell 2018).
        </p>
    </div>
</div>