---
layout: default
title: Wang Lab
subtitle: Chenfei Wang Lab at Tongji University
use-site-title: true
---

<div class="container">
    <div class="row">
        <div id="carouselExampleIndicators" class="carousel slide carousel-fade" data-ride="carousel">
          <!-- <ol class="carousel-indicators">
            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
            <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
          </ol> -->
          <div class="carousel-inner">
            <div class="carousel-item active" style="height: 25rem">
              <img src="/assets/img/lab_2024.jpg" class="d-block w-100">
            </div>
<!--             <div class="carousel-item" style="height: 25rem">
              <img src="/assets/img/lab.jpg" class="d-block w-100">
            </div> -->
          </div>
          <!-- <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
            <span class="sr-only">Previous</span>
          </a>
          <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
            <span class="carousel-control-next-icon" aria-hidden="true"></span>
            <span class="sr-only">Next</span>
          </a> -->
        </div>
    </div>

    <div class="row" style="margin-top: 3rem">
        <h2>Research interests</h2>
        <p>Our research is dedicated to developing computational algorithms and conducting large-scale data mining from single-cell multiomic data. We aim to unravel the intricate mechanisms of gene regulation and metabolic reprogramming that define cell identities. We have created methods and web resources specifically designed for single-cell multiomics, spatial multi-omics, single-cell CRISPR screen data analysis, signal enhancement, and integration. By seamlessly integrating single-cell datasets with spatial and multimodal resolution, our objective is to establish connections between changes in cell identity and disease phenotypes in various biological systems, including cancer, development, and aging.
        </p>
    </div>
    <div class="row">
        <h3>Developing intelligent algorithms for single-cell and spatial multi-omic enhancement, integration, and cross-modality integration</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/STRIDE_Cellist.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Single-cell and spatial multi-omic technologies have revolutionized our understanding of cellular heterogeneity in complex biological systems. However, corresponding analyses currently face various challenges, including inadequate resolution, coverage, and difficulty integrating and generating heterogeneous multi-modal data. To address these issues, we have developed a series of intelligent algorithms. <a href="https://github.com/wanglabtongji/STRIDE">STRIDE</a> can improve the resolution of Visium-like ST. <a href="https://github.com/wanglabtongji/Cellist">Cellist</a> enables accurate cell segmentation in high-resolution ST datasets. <a href="https://github.com/wanglabtongji/SCRIP">SCRIP</a> and <a href="https://github.com/wanglabtongji/SCRIPro">SCRIPro</a> constructs gene regulation networks (GRNs) using single-cell and spatial multiomic data. <a href="https://github.com/wanglabtongji/CCI">EvaCCI</a> evaluates cell-cell interaction algorithms, and <a href="https://github.com/wanglabtongji/SCREE">SCREE</a> analyzes multi-modal single-cell CRISPR screening data. These algorithms have enhanced the resolution of single-cell spatial omics data and provided systematic solutions to challenges such as single-cell multiomic integration, signal enhancement, and the construction of cell-fate-specific GRNs.
                </p>
            </div>
        </div>
    </div>
    <div class="row">
        <h3>Modeling cell identities using generative AI and large-scale multi-modal data</h3>
        <div class="row">
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Cell identities in a multicellular system are regulated by both intrinsic factors, including gene regulation, and extrinsic factors, such as cellular crosstalks. We have demonstrated the tight connection between intrinsic epigenetic regulations and cell-fate determination in mouse IVF and SCNT embryo development (<b>Nature, 2016; Nat. Cell Biol., 2018; Cell Stem Cell, 2018, 2022; Cell Res., 2022</b>). Currently, we are developing generative AI models pretrained on large-scale single-cell multi-modal datasets. These models aim to uncover the collaborative effects between gene regulation, cellular crosstalk, and other environmental factors, such as metabolites and mechanics, on cell identity regulation. As a pioneering work, we have introduced <a href="https://github.com/wanglabtongji/SELINA.py">SELINA</a>, which utilizes a multi-adversarial domain adaptation network to automatically annotate cell types with a large-scale human scRNA-seq reference pretrained. We hope to use generative cell identity models to provide valuable insights into the mechanisms driving cell identity transitions and to further guide and remodel the transition process. 
                </p>
            </div>
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/SELINA.png">
            </div>
        </div>
    </div>
    <div class="row">
        <h3>Discovering the diversity, plasticity, and association with clinical treatment response of cell identities in the tumor immune microenvironment</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/TabulaTIME.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Cancer arises from the evasion of immune surveillance, and the immunosuppressive tumor microenvironment has a strong impact on tumor development and therapy resistance. Our objective is to integrate single-cell and spatial multi-omics data with comprehensive bioinformatics data analysis to investigate the effects of intrinsic gene regulation and extrinsic environmental factors on altering immune cell identities in the tumor microenvironment (TME). Moreover, we aim to develop potential methods for remodeling the TME in cancer treatment. As a preliminary step, we have developed a comprehensive scRNA-seq data resource <a href="http://tisch.comp-genomics.org">TISCH</a> for analyzing gene expression and cell-type composition in the TME. Additionally, we have constructed a pan-cancer cell identity tabula <a href="http://timer2.compbio.cn/TabulaTIME">TabulaTIME</a> and discovered widespread profibrotic ecotypes that regulate tumor immunity. Currently, we are collaborating closely with oncologists and immunologists to study the mechanisms of TME evolution and immunotherapy resistance in different types of cancer (<b>Cell 2024, Genome Med., 2023, and EMBO J., 2023</b>).
                </p>
            </div>
        </div>
    </div>
  </div>
</div>


