---
layout: news
title: Software
subtitle: Softwares in Wang Lab
permalink: /software/
---

# Software
#### Single Cell Transcriptome
- [**TISCH**](http://tisch.comp-genomics.org) &nbsp;
(Tumor Immune Single-cell Hub) is a scRNA-seq database that includes over 6M human and mouse tumor scRNA-seq cells, and provides detailed cell-type annotation and gene expression visualization across different cancer types.
- [**HUSCH**](http://husch.comp-genomics.org) &nbsp;
(Human Universal Single-cell Hub) is a scRNA-seq database that includes over 3M human normal scRNA-seq cells from 41 tissue/organs, and provides detailed cell-type annotation and gene expression visualization across different tissue/organs.
- [**SELINA**](https://github.com/wanglabtongji/SELINA.py) &nbsp;
(Single-cELl Identity NAvigator) is a deep learning-based framework for cell-type annotation of human scRNA-seq data using large-scale curated references. 
- [**SCREE**](https://github.com/wanglabtongji/SCREE) &nbsp;
(Single-cell CRISPR screen data analysEs and perturbation modEling) is a workflow to perform quality control and analyses of multimodal single-cell CRISPR screen datasets.
- [**TabulaTIME**](http://timer2.compbio.cn/TabulaTIME) &nbsp;
(Tabula of pan-cancer tumor microenvironment) is a comprehensive tumor microenvironment landscape by integrating single-cell and spatial transcriptomic data.

#### Single Cell Epigenetics
- [**MAESTRO**](http://github.com/liulab-dfci/MAESTRO) &nbsp;
(Model-based Analyses of Transcriptome and Regulome) is a comprehensive workflow for integrated analysis of scRNA-seq and scATAC-seq data from multiple platforms.
- [**SCRIP**](https://github.com/wanglabtongji/SCRIP) &nbsp;
(Single Cell Regulatory network Inference using ChIP-seq and motif) is a tool for evaluating the binding enrichment of transcription regulators at the single-cell resolution based on integration of scATAC-seq and bulk ChIP-seq reference.

#### Spatial Transcriptome
- [**STRIDE**](https://github.com/wanglabtongji/STRIDE) &nbsp;
(Spatial TRanscrIptomics DEconvolution by topic modeling) is a topic-model-based deconvolution method for spatial transcriptomics by integrating with scRNA-seq.
- [**CCI**](https://github.com/wanglabtongji/CCI) &nbsp;
Benchmarking of CCI prediction tools based on the integration of single-cell and spatial transcriptomic data.