---
layout: news
title: Software
subtitle: Softwares in Wang Lab
permalink: /software/
---

# Software
#### Single Cell Transcriptome
- [**TISCH**](http://tisch.comp-genomics.org) &nbsp;
Tumor Immune Single-cell Hub (TISCH) is a scRNA-seq database that includes over 6M human and mouse tumor scRNA-seq cells, and provides detailed cell-type annotation and gene expression visualization across different cancer types.
- [**HUSCH**](http://husch.comp-genomics.org) &nbsp;
Human Universal Single-cell Hub (HUSCH) is a scRNA-seq database that includes over 2M human normal scRNA-seq cells, and provides detailed cell-type annotation and gene expression visualization across different tissue and organs.
- [**SELINA**](https://github.com/wanglabtongji/SELINA.py) &nbsp;
SELINA is a deep learning-based framework for cell-type annotation of human scRNA-seq data using large-scale curated references. 
- [**SCREEN**](https://github.com/HailinWei98/SCREEN/) &nbsp;
SCREEN (Single-cell CRISPR scREen data analyses and pErturbation modeliNg) is a workflow to perform quality control and analyses of single-cell CRISPR screen datasets.

#### Single Cell Epigenetics
- [**MAESTRO**](http://github.com/liulab-dfci/MAESTRO) &nbsp;
Model-based Analyses of Transcriptome and Regulome (MAESTRO) is a comprehensive workflow for integrated analysis of single-cell RNA-seq (scRNA-seq) and ATAC-seq (scATAC-seq) data from multiple platforms.
- [**SCRIP**](https://github.com/wanglabtongji/SCRIP) &nbsp;
SCRIP (Single Cell Regulatory network Inference using ChIP-seq) is a tool for evaluating the binding enrichment of transcription regulators at the single-cell resolution based on scATAC-seq.

#### Spatial Transcriptome
- [**STRIDE**](https://github.com/wanglabtongji/STRIDE) &nbsp;
Spatial transcriptomics deconvolution by topic modeling (STRIDE) is a topic-model-based deconvolution method for spatial transcriptomics by integrating with scRNA-seq.
- [**CCI**](https://github.com/wanglabtongji/CCI) &nbsp;
Benchmarking of CCI prediction tools based on the integration of single-cell and spatial transcriptomic data.